src/headless-cms/components/CmsPageContent.tsx

```diff
@@ -1,25 +1,12 @@
 import React from 'react';
 
-import HtmlToReact from '../../common/components/htmlToReact/HtmlToReact';
-import Container from '../../domain/app/layout/Container';
 import { PageFieldsFragment } from '../../generated/graphql-cms';
+import CmsSidebarLayout from './CmsSidebarLayout';
 
 const CmsPageContent: React.FC<{ page: PageFieldsFragment | undefined }> = ({
   page,
 }) => {
-  const title = page?.title || '';
-  const content = page?.content || '';
-
-  return (
-    <Container>
-      {page && (
-        <>
-          <h1>{title}</h1>
-          <HtmlToReact>{content}</HtmlToReact>
-        </>
-      )}
-    </Container>
-  );
+  return <CmsSidebarLayout page={page} />;
```

Comment:
Are you planning to leave this here? Do we need CmsSidebarLayout as a separate component?

Reply:
I was planning to remove it in a later PR to avoid a merge conflict.